src/main/util/helper_classes/id_counter.py

```diff
@@ -36,3 +35,11 @@ def get_item_by_id(cls: 'IdCounter', id: int) -> 'IdCounter':
         if item is None:
             log_and_raise_error(f'Item with id {id} does not exist in the class {cls.__name__}', log)
         return item
+
+    def set_item_by_id(self) -> None:
```

Comment:
You use it only in the tests. Maybe should we not have the method here? And implement it only in the tests